### PR TITLE
fedekunze/1920 add ledger installation instructions

### LINKED ISSUE #1920: Add Ledger instructions
Description:

How to install Ledger Cosmos App through ledger Live
Maybe link to the delegator guide from Gautier on the Cosmos Docs

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #1920 

**Description:**

- Add link on hardware session modal
- update `README`

screen shot 2019-02-18 at 12 01 30 pm

Thank you! 🚀

---

For contributor:

- [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [x] Reviewed `Files changed` in the github PR explorer
- [x] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/components/common/TmSessionHardware.vue [hunk 1]
```diff
@@ -22,8 +22,16 @@
           value="Connecting..."
           @click.native="setStatus('connect')"
         />
+        <p class="ledger-install">
+          Don't have the Cøsmos Ledger App yet? Install it
+          <a
+            href="https://github.com/cosmos/voyager#ledger-cosmos-app"
```
Comment (faboweb) [inline app/src/renderer/components/common/TmSessionHardware.vue]: I think this is not a good user experience. Let's add the instructions in the UI as stated in the issue.

Comment (fedekunze) [reply app/src/renderer/components/common/TmSessionHardware.vue]: you mean creating a page/modal to show the instructions?

Comment (faboweb) [reply app/src/renderer/components/common/TmSessionHardware.vue]: Yes, I think that is better UX

**Comment by faboweb** [comment]
We will do a follow up to implement the actual instructions into the UI.

**Pull request merged**